frontend/src/components/Status/index.tsx

```diff
@@ -61,3 +64,19 @@ export default component$((props: Props) => {
 		</div>
 	)
 })
+
+export const RebloggerLink = ({ account }: { account: Account | null }) => {
+	return (
+		account && (
+			<div class="flex text-wildebeest-500 py-3">
+				<p>
+					<i class="fa fa-retweet mr-3" />
+					<a class="no-underline" href={account.url}>
+						{account.display_name}
+					</a>
+					&nbsp;boosted
```

Comment:
😕

Reply:
What's wrong with this?

Reply:
`&nbsp;` feels a bit dirty, wouldn't it be better to have a span with left margin or something....

but actually `&nbsp;` does provide the same distance as a space which margin may not do precisely...

so yeah, never mind me 😁 🙇